test_data/yast/msdos/msdos_pv.yaml

```diff
@@ -0,0 +1,41 @@
+table_type: msdos
+disks:
+  - name: xvda
```

Comment:
I think you did right duplicating the data as the nested keys cannot be replaced, only first level, as far as I remember, BUT I was thinking with current implementation how we could achieve that, what about remove `disks` as you only have one and  convert name in a first level key so you can easily replaced with `!include` or override `disk_name` and `partitions`? Something like this:
```
table_type: msdos
disk_name: xvda
disk_allowed_partitions: 0.00GB
partitions:
...
```

Reply:
I will try that.

Reply:
I am a bit confused with this, as I am trying to put into the yaml schedule , besides the disk name, the partition with /boot/efi etc... but I cannot seem to find how to !include test_data file with rest of partitions, given that I cannot change the format of the test_data.

Reply:
you can change the format of the test data. that is why I suggested that might work to reorganize the keys, so if they are in first level they can be override and you safe files, of course you might need to change how to call the functions. But leave this point if you don't see a clear way to do it, it is a minor thing.

Reply:
I will check if it's possible, as I am using some of the raid tests functions and call some of the data in specific format.

Reply:
Btw, the msdos tests, do not need different yaml schedules for every architecture, only s390x is different, which I will try to set as option in same schedule, so multiple test data files will save us from many different test suites.

Reply:
yes, let's see if it works, I was referring to test data repetition.